### ISSUE TITLE
timer, and play ,rec ,stop ,block play, loop rang play

### ISSUE THREAD

**Issue by onukore** [kmatheussen/radium#352]
a big timer and count down ,and play ,rec ,stop ,block play, loop rang play  regular icon on menu bar missing  a lot, 
you're forced to do it, no DAW without it (for a pro look)

**Comment by kmatheussen** [comment]
Time was shown before 3.0, but I removed it since it seemed useless. Is it really useful for something? It's simple to put into the bottom bar, for instance.

**Comment by onukore** [comment]
yes, is usefull to know song length, and regular icon for transport to i know lot of people don't use  shortcut keyboard , another think , a mute and solo icon in track header very usefull, However on/off pan and volume , why ?
